packages/react-focus/src/components/FocusZone/FocusZone.tsx

```diff
@@ -359,14 +361,27 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
         !forceIntoFirstElement &&
         this._activeElement &&
         elementContains(this._root.current, this._activeElement) &&
-        isElementTabbable(this._activeElement)
+        isElementTabbable(this._activeElement) &&
+        (!bypassHiddenElements || isElementVisibleAndNotHidden(this._activeElement))
```

Comment:
As I understand this PR we're adding `bypassHiddenElements` as a way to opt in to a new behavior. If that is correct I think we need to change this logic so we preserve the existing behavior by default:

```suggestion
    bypassHiddenElements ? isElementVisibleAndNotHidden(this._activeElement) : true
```

This way we only opt in to the new behavior when `bypassHiddenElements` is true.

What do you think?

Reply:
Yes, I agree, that's why I have `!bypassHiddenElements || isElementVisibleAndNotHidden` that way if `bypassHiddenElements` isn't passed in (or is passed in as `false`) then the behavior will be the same as before, otherwise (e.g. `true` is passed in) is checked `isElementVisibleAndNotHidden`

Reply:
Ah, I see it now 🤦😄